### PR TITLE
Add New Featured Artist: Fuki newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2020-11-11-new-featured-artist-fuki.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+date: 2020-11-11 09:00:00 +0000
+---
+
+We're proud to welcome **Fuki** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/114/header.jpg)
+
+If you're a fan of Japanese metal or rock, you might already know **Fuki** from her time as a vocalist in bands like *LIGHT BRINGER*, *Unlucky Morpheus* and *DOLL$BOXX*. She's participated in a wide range of projects, ranging from things like radio voice-overs, voice acting, narration. She has also sung the ending themes of anime like *Kaitou Joker*, *W'z* and even the theme song for *IKEMEN REVOLUTION - Love & Magic in Wonderland*.
+
+**11** new tracks from her are now available straight from one of her most recent albums titled *Million Scarlets*, including a track featured in the osu! World Cup 2019 Finals week, and a brand new showcase set from some of the minds of the Mappers' Guild.
```
Comment (pishifat) [inline news/2020-11-11-new-featured-artist-fuki.md:13]: > one of her most recent albums

i think this is their only solo album?

Comment (Nivalyx) [reply news/2020-11-11-new-featured-artist-fuki.md:13]: also the "a brand new showcase set" here should instead be "two brand new showcase sets" i suppose...

Comment (Ephemeralis) [reply news/2020-11-11-new-featured-artist-fuki.md]: fixed both

File: news/2020-11-11-new-featured-artist-fuki.md [hunk 2]
```diff
@@ -55,7 +55,8 @@ Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/1273120) hoste
 All of the above plus a bunch more are available now over at [**Fuki**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/114), pre-timed and ready for you to spin her wide-reaching vocals into similarly impressive beatmaps.
 
 JRock and metal fans have had a *lot* of rejoicing to do with recent additions to the roster lately! Devout fans can rest assured that even more is coming soon, from names both fresh and familiar.
-
++
+++
```
Comment (peppy) [inline news/2020-11-11-new-featured-artist-fuki.md]:
```suggestion

```

**Review by peppy — CHANGES_REQUESTED** [review]

File: news/2020-11-11-new-featured-artist-fuki.md (hunk 2 above)
Comment (Ephemeralis) [reply news/2020-11-11-new-featured-artist-fuki.md]: fixed this before

**Review by peppy — APPROVED** [review]

**Pull request merged**